crates/macro/src/html_tree/html_component.rs

```diff
@@ -392,8 +395,21 @@ struct ListProps {
 impl Parse for ListProps {
     fn parse(input: ParseStream) -> ParseResult<Self> {
         let mut props: Vec<HtmlProp> = Vec::new();
-        while HtmlProp::peek(input.cursor()).is_some() {
-            props.push(input.parse::<HtmlProp>()?);
+
+        loop {
```

Comment:
Why did you change the behavior of this loop to check first for `with`?

Reply:
Because it is important to do this check
```rust
if let Some(ident) = token.ident() {
       if ident.0 == "with" {
       return Err(input.error(Props::collision_message()));
       }
 }
```
before `HtmlProp::peek(token).is_some()`

Reply:
I don't think it's important because of this line: https://github.com/yewstack/yew/pull/960/files#diff-a8c81c5f22d5371fcf4d0ff922a224b4R370

Reply:
Oh, sorry, my bad. I've fixed it.

Reply:
No problem, thanks 👍